apps/studio/components/interfaces/Database/Indexes/Indexes.tsx

```diff
@@ -221,14 +246,14 @@ const Indexes = () => {
 
       <SidePanel
         size="xlarge"
-        visible={selectedIndex !== undefined}
+        visible={!!selectedIndex}
         header={
           <>
             <span>Index:</span>
             <code className="text-sm ml-2">{selectedIndex?.name}</code>
           </>
         }
-        onCancel={() => setSelectedIndex(undefined)}
+        onCancel={() => setSelectedIndexName('')}
```

Comment:
having this be an optional type is much better than using empty string IMO. it forces the developer to think about the situation where we need to check the type and handle cases where it might be undefined. We should witch this back IMO